driver-sync/src/test/functional/com/mongodb/internal/connection/OidcAuthenticationProseTests.java

```diff
@@ -234,6 +234,8 @@ public void test2p4InvalidClientConfigurationWithCallback() {
 
     @Test
     public void test2p5InvalidAllowedHosts() {
+        assumeTestEnvironment();
```

Comment:
There is no requirement to run this test in a non-"test" environment; disabled to prevent running on GCP. Tests [passing here](https://spruce.mongodb.com/version/664e31e50c8d75000729d6ba/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC).